### DISCUSSION TITLE
How many 👧 are in the class? 🏫

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1231] [Galaxy Trivia] [answered]
In the class, 1/3 of the students brought a teddy 🐻. 
Each 👦 took 12 📗, each 👧 took 17 📕, and each teddy 🐻 took 9 📘. 
A total of 305 📚 were taken out.

**Comment by peterbaccaro** [comment, marked as answer]
There are **7** girls in the class.

**Reasoning**

number of boys = **b**
number of girls = **g**

1/3 of the students brought a teddy so number of teddy bears is **b + g / 3**

number of books boys took = **12b**
number of books girls took = **17g**
number of books teddy bears took = **9 * (b + g / 3)**
number of books teddy bears took simplified = 3(b + g)

So **12b + 17g + 3(b + g) = 305**
Simplified as **12b + 17g + 3b + 3g = 305**
Simplified as **15b + 20g = 305**
Simplified as **3b + 4g =61**

So **g = (61 - 3b) / 4**

number of boys (**b**) and girls (**g**) needs to be a whole number greater than 0 so you can put values for **b** and see if **g** is a whole number.

The values of b where g is a whole number are:
- If **b** is 3 then **g** = 52/4 = 13
- If **b** is 7 then **g** = 49/4 = 10
- If **b** is 11 then **g** = 46/4 = 7
- If **b** is 15 then **g** = 43/4 = 4
- If **b** is 19 then **g** = 40/4 = 10

number of bears also needs to be a whole number greater than 0.

- If **b** is 3 and **g** = 13 then teddy bears would be = 16/3 = 5.33
- If **b** is 7 and **g** = 10 then teddy bears would be = 17/3 = 5.66
- If **b** is 11 and **g** = 7 then teddy bears would be = 18/3 = **6**
- If **b** is 15 and **g** = 4 then teddy bears would be = 19/3 = 6.33
- If **b** is 19 and **g** = 10 then teddy bears would be = 20/3 = 6.66

Since only one of the above options results in a whole number of teddy bears we have the answer.

So there are 11 boys, **7 girls** and 6 teddy bears in the class.
